### ISSUE TITLE
Add support for KeyboardInterrupt in REPL

### ISSUE THREAD

**Issue by YashSinha1996** [cknd/stackprinter#5]
On keyboard interrupt in REPL, it raises the following stacktrace:

```
Error in sys.excepthook:
Traceback (most recent call last):
  File "/datadrive/product_type_article/venv/lib/python3.7/site-packages/stackprinter/__init__.py", line 245, in hook
    show(args, **kwargs)
  File "/datadrive/product_type_article/venv/lib/python3.7/site-packages/stackprinter/__init__.py", line 23, in show_or_format
    return f(thing, *args, **kwargs)
  File "/datadrive/product_type_article/venv/lib/python3.7/site-packages/stackprinter/__init__.py", line 163, in show
    print(format(thing, **kwargs), file=file)
  File "/datadrive/product_type_article/venv/lib/python3.7/site-packages/stackprinter/__init__.py", line 23, in show_or_format
    return f(thing, *args, **kwargs)
  File "/datadrive/product_type_article/venv/lib/python3.7/site-packages/stackprinter/__init__.py", line 142, in format
    "a frame or a thread object." % repr(thing))
ValueError: Can't format (<class 'KeyboardInterrupt'>, KeyboardInterrupt(), None). Expected an exception instance, sys.exc_info() tuple,a frame or a thread object.

Original exception was:
KeyboardInterrupt
```

I would love to contribute to help out, but I am not sure what the behaviour should be in this case? 

Awesome project BTW.

**Comment by cknd** [comment]
Thanks! 
Can you post your python version? I can't reproduce it on 3.5.2. Though I get a similar problem on syntax errors in the REPL and might have an idea to fix both. Looks like the traceback is None for that exception, which should be easy to deal with (= just print the exception)

**Comment by cknd** [comment]
https://github.com/cknd/stackprinter/pull/6

**Comment by YashSinha1996** [comment]
Hey @cknd I am on 3.7.2.

**Comment by cknd** [comment]
This should be fixed on 0.2.0. Can you confirm?

**Comment by YashSinha1996** [comment]
Hey @cknd thanks a lot. Fixed!
